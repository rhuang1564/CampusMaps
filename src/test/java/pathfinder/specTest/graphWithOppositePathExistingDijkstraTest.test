# Find a path between node and another node
# but only edge going opposite way Dijkstra's algorithm
# in a single edged graph

CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B 1.0

FindPath graph B A