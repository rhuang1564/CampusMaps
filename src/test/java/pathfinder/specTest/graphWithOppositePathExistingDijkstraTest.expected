# Find a path between node and another node
# but only edge going opposite way Dijkstra's algorithm
# in a single edged graph

created graph graph

added node A to graph
added node B to graph

added edge 1.000 from A to B in graph

path from B to A:
no path found